internal/manifest/oci_index.go

```diff
@@ -79,6 +82,20 @@ func (index *OCI1IndexPublic) UpdateInstances(updates []ListUpdate) error {
 	return index.editInstances(editInstances)
 }
 
+func annotationsToCompressionAlgorithmNames(annotations map[string]string) []string {
+	result := []string{}
+	for k, v := range annotations {
+		if k == OCI1InstanceAnnotationCompressionZSTD && v == OCI1InstanceAnnotationCompressionZSTDValue {
```

Comment:
This is just `if annotations[…Zstd] == …ZstdValue`.

Reply:
Done.